apps/web/app/[locale]/page-component.tsx

```
@@ -35,163 +29,144 @@ import { headerTabs } from '@app/stores/header-tabs';
+		{ title: t(`common.${view}`), href: `/` }
+	];
+	const { online } = useNetworkState();
+	useEffect(() => {
+		if (view == IssuesView.KANBAN && path == '/') {
+			setView(IssuesView.CARDS);
+		}
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, [path, setView]);
+
+	React.useEffect(() => {
+		window && window?.localStorage.getItem('conf-fullWidth-mode');
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)